bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/adapters/TestColumnDescriptorAdapter.java

```diff
@@ -191,10 +191,8 @@ public void badExpressionLessThan(){
 
   @Test
   public void testBlankExpression(){
```

Comment:
I have 1 and 2.  The result from #2 is just an empty string, so I don't think that #3 is necessary.

Renaming the test method and comiting.